app/uk/gov/hmrc/minorentityidentificationfrontend/services/AuditService.scala

```diff
@@ -35,60 +35,69 @@ class AuditService @Inject()(auditConnector: AuditConnector,
       journeyConfig <- journeyService.getJourneyConfig(journeyId, authInternalId)
       optUtr <- storageService.retrieveUtr(journeyId)
       optOverseasTaxIdentifiers <- storageService.retrieveOverseasTaxIdentifiers(journeyId)
-    } yield journeyConfig.businessEntity match {
-      case OverseasCompany =>
-        val optUtrBlock = optUtr match {
-          case Some(utr: Ctutr) => Json.obj("userCTUTR" -> utr.value, "cTUTRMatch" -> false)
-          case Some(utr: Sautr) => Json.obj("userSAUTR" -> utr.value, "sautrMatch" -> false)
-          case None => Json.obj()
+      optRegistrationStatus <- storageService.retrieveRegistrationStatus(journeyId)
+    } yield {
+      val optUtrBlock = optUtr match {
```

Comment:
optUtrBlock and overseasIdentifiersBlock are currently only used for OverseasCompanies. So perhaps their definition should be inside the OverseasCompanies case section.